src/schema/v2/notifications/index.ts

```diff
@@ -104,7 +105,20 @@ export const NotificationType = new GraphQLObjectType<any, ResolverContext>({
     artworksConnection: {
       type: artworkConnection.connectionType,
       args: pageable(),
-      resolve: async ({ object_ids: ids }, args, { artworksLoader }) => {
+      resolve: async (
+        { notificationType, object_ids: ids },
+        args,
+        { artworksLoader, mePartnerOfferLoader }
+      ) => {
+        if (notificationType == "PartnerOfferCreatedActivity") {
```

Comment:
As above, I _think_ this isn't needed as `object_ids` should already contain the artwork id's.